indicators/discord-fake-404-kit.yml

```diff
@@ -0,0 +1,25 @@
+title: Discord Fake 404 Kit
+description: |
+  Detects a fake 404 page that is used on Discord phishing templates. These sites often have hidden paths of /login with a fake Discord login.
+references:
+    - https://urlscan.io/result/43143a94-ee73-46da-a877-e0f0e5eb5d0c/
+    - https://urlscan.io/result/d360ec5c-ec44-484e-8bf5-7ad3a296adbb/
+    - https://urlscan.io/search/#page.title%3A%22Page%20Not%20Found%20%7C%20404%22
+
+detection:
+
+  pageTitle:
+    title: "Page Not Found | 404"
+
+  pageHTML:
+    html|contains|all:
+      - "<head><title>Page Not Found | 404</title>"
```

Comment:
Given the rule already detects the title, I think this is redundant.

Reply:
Good point, thanks for pointing that out.

Reply:
Fixed.